modules_scripts/mod_pvp_titles/src/mod_pvp_titles.cpp

```diff
@@ -0,0 +1,145 @@
+#include "ScriptMgr.h"
+#include "Log.h"
+#include "Player.h"
+#include "Chat.h"
+#include "Common.h"
+#include "World.h"
+#include "WorldSession.h"
+#include <unordered_map>
+
+enum Ranks
+{
+	RANK_1	= 50,
```

Comment:
Would it be a good idea to set this database driven for config? :) sWorld.GetModuleIntConf("PvPTitle.Rank1", 50) ect :D